smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/PackageManifestGenerator.kt

```diff
@@ -27,49 +30,64 @@ class PackageManifestGenerator(val ctx: ProtocolGenerator.GenerationContext) {
                     writer.write(".library(name: \$S, targets: [\$S])", ctx.settings.moduleName, ctx.settings.moduleName)
                 }
 
-                val externalDependencies = dependencies.filter { it.getProperty("url", String::class.java).get().isNotEmpty() }
-                val dependenciesByURL = externalDependencies.distinctBy { it.expectProperty("url", String::class.java) }
+                val externalDependencies = dependencies.filter {
+                    it.getProperty("url", String::class.java).getOrNull() != null ||
+                        it.getProperty("scope", String::class.java).getOrNull() != null
+                }
+                val dependenciesByURL = externalDependencies.distinctBy {
+                    it.getProperty("url", String::class.java).getOrNull()
+                        ?: "${it.getProperty("scope", String::class.java).get()}.${it.packageName}"
+                }
 
                 writer.openBlock("dependencies: [", "],") {
-                    dependenciesByURL.forEach { dependency ->
-                        writer.openBlock(".package(", "),") {
-
-                            val localPath = dependency.expectProperty("localPath", String::class.java)
-                            if (localPath.isNotEmpty()) {
-                                writer.write("path: \$S", localPath)
-                            } else {
-                                val dependencyURL = dependency.expectProperty("url", String::class.java)
-                                writer.write("url: \$S,", dependencyURL)
-                                writer.write("from: \$S", dependency.version)
-                            }
-                        }
-                    }
+                    dependenciesByURL.forEach { writePackageDependency(writer, it) }
                 }
 
                 val dependenciesByTarget = externalDependencies.distinctBy { it.expectProperty("target", String::class.java) + it.packageName }
```

Comment:
"Dependencies by target" are written to the service client target's dependencies section.